runtime/dali/src/lib.rs

```diff
@@ -1207,7 +1207,7 @@ construct_runtime!(
 
 		Tokens: orml_tokens::{Pallet, Call, Storage, Event<T>} = 51,
 		Oracle: oracle::{Pallet, Call, Storage, Event<T>} = 52,
-		CurrencyFactory: currency_factory::{Pallet, Storage, Event<T>} = 53,
+		CurrencyFactory: currency_factory = 53,
```

Comment:
Changed with the new dot version?

Reply:
we forget to add call. but with 9.22 it all auto.